### ISSUE TITLE
Local module: file does not exist

### ISSUE THREAD

**Issue by regisphilibert** [gohugoio/hugo#6722]
Reading this [thread](https://discourse.gohugo.io/t/modules-error-file-does-not-exist/20794) I think the setup form this test [repo](https://github.com/regisphilibert/hugo-issue-6722) should work. Yet I get the following error: 
```
Error: add site dependencies: load resources: loading templates: walk: open "debug" (""): file does not exist
```

What it does is try and load a module whose files are stored locally (in the themes directory as mentioned in the doc) and use the mounts directive to assign its directory to Hugo's filesystem.

```
module:
  imports:
    - path: imgix
```

And from the module config.yaml:
```
module:
  hugoVersion:
    extended: false
    min: "0.62.2"
  mounts:
    - source: debug
      target: layouts/partials/debug
```

**Comment by bep** [comment]
I don't remember any tests for custom mounts for /themes folders, so I guess that is not supported.

Why not move your debug folder below layouts/partials where it belongs. I don't see what you save by your current setup.

**Comment by regisphilibert** [comment]
> Why not move your debug folder below layouts/partials where it belongs.

Then it's just a regular theme, bound to follow a Hugo Project's file structure. I wish I could have a module (which will later be move to its own repo after development) from which I can handpick various files/directory.
I was trying to replicate what you achieve in https://github.com/bep/portable-hugo-links

**Comment by regisphilibert** [comment]
Closed in favor of #6730
